File: backend/team6/src/main/java/programmers/team6/domain/auth/config/JwtConfiguration.java
package programmers.team6.domain.auth.config;

import org.springframework.boot.context.properties.ConfigurationProperties;

import lombok.extern.slf4j.Slf4j;

@Slf4j
@ConfigurationProperties(prefix = "jwt")
public record JwtConfiguration(
	String secret,
	long accessTokenExpiration,
	long refreshTokenExpiration,
	String header,
	String prefix
) {
}

File: backend/team6/src/main/java/programmers/team6/domain/auth/token/JwtConfiguration.kt
package programmers.team6.domain.auth.token

import org.springframework.boot.context.properties.ConfigurationProperties


@ConfigurationProperties(prefix = "jwt")
data class JwtConfiguration(
	val secret: String,
	val accessTokenExpiration: Long,
	val refreshTokenExpiration: Long,
	val header: String,
	val prefix: String
)
